website/static/js/addonNames.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/*
+ A simple table of addon_short_name: addon_full_name pairs
+*/
+module.exports = {
```

Comment:
All addon short and full names are already configured on the backend. How about serializing both names and passing them to the `ExternalAccount` constructor instead of duplicating them in the js?